massa-execution-worker/src/interface_impl.rs

```diff
@@ -571,7 +571,6 @@ impl Interface for InterfaceImpl {
             validity_start: Slot::new(validity_start.0, validity_start.1),
             validity_end: Slot::new(validity_end.0, validity_end.1),
             max_gas,
-            gas_price: Amount::from_raw(gas_price),
```

Comment:
* this should be replaced by `fee: Amount`
* massa-sc-runtime should also be updated